Development/#3 Post form from React frontend to Django Restful API-20210930T183306Z-001/#3 Post form from React frontend to Django Restful API/frontend/src/components/leadFormContainerView.jsx

```diff
@@ -18,7 +18,13 @@ class LeadFormContainerView extends Component {
     };
   }
 
-  handleSubmit() {}
+  handleSubmit() {
+    axios
+      .post("/api/leads/",this.state.lead)

```

Comment:
Could you use object destructuring?

Reply:
I think lead object is defined above so its fine. Or should I change it?

Reply:
Take a look at some other functions in the react files. Can you store the lead object as `const { lead } = this.state;` for consistency?

Reply:
@AnirudhAchal Done Please review